### ISSUE TITLE
NesT question - aggregation maxpool - 3x3 kernel, stride 2 or 2x2 kernel, stride 2?

### ISSUE THREAD

**Issue by lessw2020** [lucidrains/vit-pytorch#121]
Hi @lucidrains,
I'm working with the NesT already (thanks again!) but wondering on one minor point re: Aggregation block:
~~~
def Aggregate(dim, dim_out):
    return nn.Sequential(
        nn.Conv2d(dim, dim_out,3, padding=1),
        channelNorm(dim_out),
       nn.MaxPool2d(2)  # < -- **_should this be  nn.MaxPool2d(3,2)**  ??  
    )
  ~~~
Should it be nn.MaxPool2d(3,2) or nn.MaxPool2d(2)?  
Current code atm is nn.MaxPool2d (2), but in looking at the paper I interpret it as 3x3 kernel, stride 2 max pooling, or nn.MaxPool2d(3,2).
I'll try and test with both but wondering if you can advise?

Thanks again for this great impl!

**Comment by lucidrains** [comment]
@lessw2020 yes, you are right Less! Thank you for catching this bug :pray: https://github.com/lucidrains/vit-pytorch/releases/tag/0.19.2

**Comment by lessw2020** [comment]
@lucidrains great, thanks for confirming and updating Phil!
